docs/admin/maintenance/noble_migration_prep.rst

```diff
@@ -0,0 +1,128 @@
+Preparing for the Noble migration
+=================================
+
+In 2025, SecureDrops will need to be upgraded to the newer Ubuntu Noble (24.04)
+operating system. This process will be more straightforward than in the past
+as there will be semi-automated and fully automated upgrade processes.
+
+At this time, the current focus is on ensuring all SecureDrop servers are in a good state
+to be ready for the migration. SecureDrop will automatically check some conditions, and report
+via an alert in the Journalist Interface and OSSEC alerts if there are issues.
+
+Getting more details
+--------------------
+
+If you see the alert in the Journalist Interface or receive an OSSEC alert, you'll
+need to :doc:`SSH to the Application and Monitor Servers <../installation/test_the_installation>`
+to get more information on what specifically is failing.
+
+Once logged in, run:
+
+.. code:: sh
+
+   sudo securedrop-noble-migration-check
+
+It will display a number of checks and whether they are failing.
+Steps to address each issue are listed below. If you are unsure what to do,
+please :ref:`contact Support <getting_support>`. It is safe to run this command
+multiple times, e.g. if you resolved an issue and want to see that it is fixed.
+
+For example:
+
+.. code:: sh
+
+    $ sudo securedrop-noble-migration-check
```

Comment:
This output is based on the changes I've proposed in <https://github.com/freedomofpress/securedrop/pull/7369>.